test/IECore/VectorData.py

```diff
@@ -58,6 +58,18 @@ def testConstructors(self):
 		v4 = _vector(10)
 		self.assertEqual( len( v4 ), 10 )
 
+	def testSerialising(self):
+		"""Test serialising and deserialising """
+		_vector = self.vectorFactory
+		_c = self.valueFactory
+		v1 = _vector([_c(23), _c(4), _c(34), _c(12)])
+		w = IECore.Writer.create(v1, '/tmp/vectorTest.cob')
```

Comment:
We should be able to avoid the temp files using `MemoryIndexeIO` (see https://github.com/ImageEngine/cortex/blob/master/test/IECore/VectorData.py#L1278-L1283)